back-end/app/Repositories/Implementations/UserImpl.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace App\Repositories\Implementations;
+
+use App\Models\User;
+use App\Repositories\Interfaces\UserInt;
+
+class UserImpl extends CommonImpl implements UserInt
+{
+    /** @var User $user */
+    private $user;
+
+    public function __construct (User $user)
+    {
+        parent::__construct($user);
+
+        $this->user = $user;
+    }
+
+    public function login()
+    {
+
+    }
+
+    public function passwordrecover()
```

Comment:
Mi pareva di averlo già scritto da qualche parte, ma userei nome in camelCase, anche sotto sul reset.

Reply:
Lascia perdere le cose su user